docs/installation/installing-octopus/automating-installation.md

```diff
@@ -0,0 +1,108 @@
+---
+title: Automating Octopus installation
+description: Information on how to install and configure an Octopus server in a fully automated way from the command line.
+---
+
+## Octopus installers
+
+Octopus comes in a MSI that can be deployed via group policy or other means.
+
+The latest Octopus MSI can always be [downloaded from the Octopus Deploy downloads page](https://octopus.com/downloads).
+
+Permalinks to always get the latest MSIs are:
+
+- 32-bit: [https://octopus.com/downloads/latest/WindowsX86/OctopusServer](https://octopus.com/downloads/latest/WindowsX86/OctopusServer)
+- 64-bit: [https://octopus.com/downloads/latest/WindowsX64/OctopusServer](https://octopus.com/downloads/latest/WindowsX64/OctopusServer)
+
+Automating the installation of Octopus Server is a three step process.
+
+### 1. Install MSI on a temporary machine interactively
```

Comment:
After discussing with @tothegills we have decided to leave it this way because of the different permutations of configuration, would make a very long and hard to follow script